flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
![image](https://user-images.githubusercontent.com/8841466/164455056-a5b071b4-4496-44aa-b732-658f28fea0a0.png)

Reply:
Also I find it funny how selective you are with complaining about the flathub.json files in other PRs you looked at you ignored the exact same flathub file.

Reply:
@Mar0xy, 

Earlier @hfiguiere was under the impression that the builds don't run on aarch64 anymore so he suggested the file to be removed. 
The [build clearly shows that it is being built on aarch64](https://flathub.org/builds/#/builders/7/builds/6760) so I agree that restoration of the file makes sense.

As I pointed out earlier the [official docs](https://github.com/flathub/flathub/wiki/App-Maintenance) also encourage the use of said file.

Reply:
not it's not funny. I have been complaining about x86_64 only sumbissions since forever. But if you want to pick on it, you're welcome and I will tell you why I didn't:

1. proprietary
2. wine
3. Flutter

For starter. There are others.

Here I'm noting that a MIT licensed software using a cross platform solution (Electron) that DO HAVE support for aarch64 isn't packaged on aarch64.

Reply:
arm != aarch64.

The former has been dropped two years ago.

My disappointement is the making of a package that is x86_64 when there doesn't seem to be a justification for it since the software could be built from source.

Reply:
> a package that is x86_64 **only** when there doesn't seem to be a justification for it

Actually I just tried building it for aarch64 without avail as we use castlabs electron with widevine for HiFi quality support and castlabs electron doesn't have aarch64 as there is no officially supported widevine cdm for it [Mention by a castlabs dev](https://github.com/castlabs/electron-releases/issues/114#issuecomment-968842678)